Gopkg.toml

```diff
@@ -1,17 +1,15 @@
-required = ["github.com/Masterminds/semver"]
```

Comment:
Why was this `required` to begin with? I would have thought the direct dependency (below) would have been enough, so I'm not sure why it was needed before and not anymore?